openshift/templates/api/api-deploy.yaml

```diff
@@ -107,7 +34,6 @@ objects:
             - podSelector:
                 matchLabels:
                   role: ${PUBLIC_FRONTEND_ROLE}
-                  app: ${APP_NAME}${SUFFIX}
```

Comment:
I don't think this should have been removed.  This directly affects network policies and their resolution.